seqr/urls.py

```diff
@@ -178,6 +179,7 @@
     'individual/(?P<individual_guid>[\w.|-]+)/update_case_review_discussion': update_case_review_discussion,
     'individual/(?P<individual_guid>[\w.|-]+)/update_case_review_status': update_case_review_status,
     'individual/(?P<individual_guid>[\w.|-]+)/rna_seq_data': get_individual_rna_seq_data,
+    'individual/(?P<individual_guid>[\w.|-]+)/rna_seq_splice_data': get_individual_rna_seq_splice_data,
```

Comment:
We could use one API for both expression and splicing outlier data. But I need to figure it out.

Reply:
This data will be displayed on the same page now, so it makes sense to have a single endpoint to fetch all the data for the page. The only question is if there are any cases where we use this endpoint not for that page?

Reply:
In that case we should modify the existing `rna_seq_data` endpoint to return all the data and not add a new endpoint